app/services/srch_scope.rb

```diff
@@ -3,12 +3,12 @@ class SrchScope
   def self.find_users(input, limit)
     if ActiveRecord::Base.connection.adapter_name == 'Mysql2'
       User.search(input)
-          .order('id DESC')
-          .where(status: 1)
+          .order('id DESC') # do we need this line?
```

Comment:
This orders returned profiles roughly by creation date, since the `id` column is auto-incrementing. If we're ordering later, we can remove it from here.